Chapter-4-applying-tactical-domain-driven-design/Fitnet.Contracts/Src/Fitnet.Contracts.Core.UnitTests/SignContract/SignContractTests.cs

```diff
@@ -20,10 +20,10 @@ public sealed class SignContractTests
             .PreparedAt(preparedAt);
 
         // Act
-        var bindingContract = contract.Sign(signedAt, fakeNow);
+        var signResult = contract.Sign(signedAt, fakeNow);
```

Comment:
If you want to go this way, then stick to convention with variable name "result" :)

Reply:
fixed. Thanks!